Multisig/Features/Connect to Wallet/Logic/WCAppRegistryRepository.swift

```diff
@@ -46,11 +46,11 @@ class WCAppRegistryRepository {
         guard let cdEntries = (try? CDWCAppRegistryEntry.getAll()) else {
             return
         }
-        var keysByEntry: [String: KeyInfo] = [:]
+        var keysByEntry: [String: NSSet] = [:]
```

Comment:
can we be more specific here and restrict type of collection?

Reply:
it's an objective-c NSSet from the core data, I think it's OK